COPYRIGHT

```diff
@@ -0,0 +1,1090 @@
+
+
+
+%%%%%%%%%
+
+luasyslog
+
+http://lua.net-core.org/sputnik.lua?p=Telesto:About
+
+
+
+%%%%%%%%%
+
+luatz
+
+https://github.com/daurnimator/luatz
+https://github.com/daurnimator/luatz/blob/master/COPYING
+
+The MIT License (MIT)
+
+Copyright (c) 2013-2017 Daurnimator
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of
+this software and associated documentation files (the "Software"), to deal in
+the Software without restriction, including without limitation the rights to
+use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+the Software, and to permit persons to whom the Software is furnished to do so,
+subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+%%%%%%%%%
+
+mimetypes
+
+https://bitbucket.org/leafstorm/lua-mimetypes/
+https://bitbucket.org/leafstorm/lua-mimetypes/src/eb83f676db53ebce7bae5d3e800c72477660f298/LICENSE?at=default&fileviewer=file-view-default
+
+Copyright (c) 2011 Matthew "LeafStorm" Frazier
+
+Permission is hereby granted, free of charge, to any person
+obtaining a copy of this software and associated documentation
+files (the "Software"), to deal in the Software without
+restriction, including without limitation the rights to use,
+copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the
+Software is furnished to do so, subject to the following
+conditions:
+
+The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+OTHER DEALINGS IN THE SOFTWARE.
+
+======
+
+In addition, the MIME types contained in the Software were
+originally obtained from the Python 2.7.1 ``mimetypes.py`` module,
+though they have been considerably modified and augmented.
+Said file was made available under the Python Software Foundation
+license (http://python.org/psf/license/).
+
+%%%%%%%%%
+
+Open-Resty
```

Comment:
Could we spell OpenResty correctly? :/

Reply:
That is how the library identifies itself, it seems.

Reply:
"OpenResty®", which is how they call it in every single use of the name in their front page :laughing:

Reply:
It is registered since last year or so, after all :)

Reply:
Where? I could not find any occurence of OpenResty spelled "Open-Resty". This is certainly not how it is generally spelled (in fact, I have never seen it spelled that way, ever). The copyright is in fact attributed to "Yichun "agentzh" Zhang, OpenResty Inc." (no dashes).